sdk/eventhub/Azure.Messaging.EventHubs/src/Azure.Messaging.EventHubs.csproj

```diff
@@ -70,7 +70,7 @@
     </EmbeddedResource>
 
     <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" PrivateAssets="All" />
-    <PackageReference Include="Microsoft.IdentityModel.Clients.ActiveDirectory" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.IdentityModel.Clients.ActiveDirectory" PrivateAssets="All" VersionOverride="[3.14.2, 4.0.0)" />
```

Comment:
Is there a reason this needs to override and cannot work with 4.5.1?

Reply:
Yes;  the management library for Event Hubs has a version dependency on 3.14.2;  I was unable to get a clean build without allowing it.    I'm open to suggestions, if you've got any.  I wasn't able to figure a cleaner way.

Reply:
I think we want to be careful about pinning our track 2 library to 3.14.2 and not work with the latest 4.5.1. If it is just because our event hub tests for management cannot consume this library then we should fix those tests. I expect we don't care if those tests are consuming the latest track 2 library.

Reply:
In the case of Event Hubs, this is a short term dependency only needed until the embedded track one client goes away (planned before GA).    Afterwards, it has no dependency on IdentityModel.